assets/app/scripts/controllers/services.js

```diff
@@ -24,7 +25,24 @@ angular.module('openshiftConsole')
       $scope.emptyMessage = "No services to show";
       updateFilterWarning();
       console.log("services (subscribe)", $scope.unfilteredServices);
-    }));    
+    }));
+
+    watches.push(DataService.watch("routes", $scope, function(routes){
+        $scope.routes = sortRoutes(routes._data);
```

Comment:
The `.by` function does not support multiple entries like we will have for routes (1 service name, many routes).  Will make the rest of the updates asap.  Thanks for the reviews!